internal/jobcontainers/jobcontainer.go

```diff
@@ -125,52 +112,53 @@ func Create(ctx context.Context, id string, s *specs.Spec) (_ cow.Container, err
 	}
 	job, err := jobobject.Create(ctx, options)
 	if err != nil {
-		return nil, errors.Wrap(err, "failed to create job object")
+		return nil, nil, errors.Wrap(err, "failed to create job object")
 	}
 
 	// Parity with how we handle process isolated containers. We set the same flag which
 	// behaves the same way for a silo.
 	if err := job.SetTerminateOnLastHandleClose(); err != nil {
-		return nil, errors.Wrap(err, "failed to set terminate on last handle close on job container")
+		return nil, nil, errors.Wrap(err, "failed to set terminate on last handle close on job container")
 	}
 	container.job = job
 
-	var path string
+	r := resources.NewContainerResources(id)
 	defer func() {
 		if err != nil {
 			container.Close()
-			if path != "" {
-				_ = removeSandboxMountPoint(ctx, path)
-			}
+			_ = resources.ReleaseResources(ctx, r, nil, true)
 		}
 	}()
 
-	limits, err := specToLimits(ctx, id, s)
-	if err != nil {
-		return nil, errors.Wrap(err, "failed to convert OCI spec to job object limits")
+	sandboxPath := fmt.Sprintf(sandboxMountFormat, id)
+	if err := mountLayers(ctx, s, sandboxPath); err != nil {
+		return nil, nil, errors.Wrap(err, "failed to mount container layers")
 	}
+	container.sandboxMount = sandboxPath
 
-	// Set resource limits on the job object based off of oci spec.
-	if err := job.SetResourceLimits(limits); err != nil {
-		return nil, errors.Wrap(err, "failed to set resource limits")
+	layers := layers.NewImageLayers(nil, "", s.Windows.LayerFolders, sandboxPath, false)
+	r.SetLayers(layers)
+
+	volumeGUIDRegex := `^\\\\\?\\(Volume)\{{0,1}[0-9a-fA-F]{8}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{12}(\}){0,1}\}(|\\)$`
+	if matched, err := regexp.MatchString(volumeGUIDRegex, s.Root.Path); !matched || err != nil {
+		return nil, nil, fmt.Errorf(`invalid container spec - Root.Path '%s' must be a volume GUID path in the format '\\?\Volume{GUID}\'`, s.Root.Path)
+	}
+	if s.Root.Path[len(s.Root.Path)-1] != '\\' {
+		s.Root.Path += `\` // Be nice to clients and make sure well-formed for back-compat
 	}
```

Comment:
do we still need this?

Reply:
Yes, the call fails otherwise. HCS returns the volume without one :(

Reply:
but don't we do the same thing in mountSandboxVolume already?

Reply:
Oop didn't realize this was present somewhere else still, thought this WAS about the one in mountSandboxVolume. Thanks will fix!